word_detector/__init__.py

```diff
@@ -119,6 +119,7 @@ def _cluster_lines(detections: List[DetectorRes],
             union = a.h + b.h - intersection
             iou = np.clip(intersection / union if union > 0 else 0, 0, 1)
             dist_mat[i, j] = 1 - iou  # Jaccard distance is defined as 1-iou
+            #dist_mat[j, i] = dist_mat[i, j] # the matrix is symetric
```

Comment:
```suggestion
            dist_mat[i, j] = dist_mat[j, i] = 1 - iou  # Jaccard distance is defined as 1-iou
```

You can use "suggestion" in reviews. I think it makes things easier to understand
![image](https://user-images.githubusercontent.com/4628382/148181751-4edc828f-e716-4ff3-8bc8-c57d838cc7b5.png)